charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,69 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Data Processing Service is a service that executes data processing pipelines containing scientific analysis on scientific data in the Data Coordination Platform (DCP).  
+
+## Objectives
+
+The Data Processing Service develops the infrastructure to run pipelines reliably and at-scale for the DCP. The team implementing the service also operates the pipeline infrastructure for the Human Cell Atlas project. 
+
+## Definitions
+
+__pipeline:__ A collection of one or more functional tasks that operate on input data and, from that, make an output that is transformed input data, or features generated to interpret the input data.
+
+## In-scope
+
+* Creating and maintaining the Data Processing Service  
+
+### Operations
+
+* Operating the instance of the HCA DCP Data Processing Service
+* Initiation and management of the execution of all data processing pipelines that run on scientific data in the HCA DCP  
+* Establishing and operating automated practices around engineering testing of pipelines executed in the HCA DCP Date Processing Service   
+
+### Documentation
+
+* Creating and outputting metadata on the operations done to the data in accordance with the HCA DCP Metadata Schema  
+* Documentation on the Data Processing Service and guides on using the service  
+* Creating documentation and scripting around the deployment of the Data Processing Service
+
+## Out-of-scope
+
+* Standing up or operating other instances of the Data Processing Service for other projects  
```

Comment:
Is this needed? The charter is in the `humancellatlas` github org, so I can't envision anyone thinking that through this charter, you're agreeing to process data or operate a service for another group.

Reply:
I believe there will be a time where the products and projects are less confounded and this will be important but I am happy to remove this, for now. The charters can be updated of course.